lib/EventSystemEpic/ProjectFeature/Model/ignore.js

```diff
@@ -0,0 +1,12 @@
+"use babel";
+// @flow
+
+const nodeModules = "node_modules/";
+const tmpDir = "tmp/";
+const tmpExt = ".tmp";
+const gitExt = ".git";
+
+const paths = [nodeModules, tmpDir, tmpExt, gitExt];
```

Comment:
Why not putting the values directly into the array ?

Reply:
C L A R I T Y